addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanRule.java

```diff
@@ -747,9 +698,8 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
             }
         } catch (Exception e) {
             logger.error(
-                    "An error occurred while checking a URI [ "
-                            + msg.getRequestHeader().getURI().toString()
-                            + " ] for cacheability",
+                    "An error occurred while checking a URI [{}] for cacheability",
+                    msg.getRequestHeader().getURI().toString(),
```

Comment:
do we need the `toString()` here?

Reply:
No. However, if that's the only occurrence I think we should let it go. The submitter simply re-used what was already there (L751).

Reply:
Seems like it's the only occurrence.

Reply:
Apologies for being nit-picky but the pscanrulesAlpha was skipped in https://github.com/zaproxy/zap-extensions/pull/2778 because of this particular PR 😅. I'm okay either way though, just saying...

Reply:
Good catch, @pranavsaxena17 mind tweak that?

Reply:
It's upto you guys, I can make the change immediately if it's deemed necessary.

Reply:
@ricekot good point.

Reply:
@pranavsaxena17 if you don't mind that would be wonderful!

Reply:
Done

Reply:
Thank you!

Reply:
@ricekot On an unrelated note, is there any way to contact you personally (I tried on slack but you seem unresponsive over there)? There was something I needed help with.

Reply:
@pranavsaxena17 replied on slack.